### ISSUE TITLE
localNotification together with badge

### ISSUE THREAD

**Issue by mixersoft** [katzer/cordova-plugin-badge#23]
I had the local-notification plugin installed and it was working fine

     cordova plugin add de.appplant.cordova.plugin.local-notification

then I tried to install badge because I was not able to clear the badge icon

     cordova plugin add de.appplant.cordova.plugin.badge

Now I'm getting this error when I try to build in xcode

    duplicate symbol _UIApplicationRegisterUserNotificationSettings in:
    .../Debug-iphonesimulator/On-the-Go.build/Objects-normal/x86_64/AppDelegate+APPBadge.o
    .../Debug-iphonesimulator/On-the-Go.build/Objects normal/x86_64/AppDelegate+APPLocalNotification.o
    ld: 1 duplicate symbol for architecture x86_64
    clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by katzer** [comment]
Has been fixed on booth plugins

**Comment by mixersoft** [comment]
thanks!

**Comment by skotturi** [comment]
I'm still having this error:

```
duplicate symbol _UIApplicationRegisterUserNotificationSettings in:
    /Users/user/Library/Developer/Xcode/DerivedData/nots-gddjalimveiqomaolqusvrsjxvne/Build/Intermediates/nots.build/Debug-iphonesimulator/nots.build/Objects-normal/x86_64/AppDelegate.o
    /Users/user/Library/Developer/Xcode/DerivedData/nots-gddjalimveiqomaolqusvrsjxvne/Build/Intermediates/nots.build/Debug-iphonesimulator/nots.build/Objects-normal/x86_64/AppDelegate+APPLocalNotification.o
ld: 1 duplicate symbol for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by toddhalfpenny** [comment]
I'm seeing this with Badge (v0.6.4) and localNotification (v0.8.1)  - I wanted to try with 0.8.2dev but couldn't find that version anywhere. Does anyone know how I can install this version please?

**Comment by t1tlehto** [comment]
I'm also seeing this with this combination:
de.appplant.cordova.common.registerusernotificationsettings 1.0.1 "RegisterUserNotificationSettings"
de.appplant.cordova.plugin.badge 0.7.1 "Cordova Badge Plugin"
de.appplant.cordova.plugin.local-notification 0.8.2dev "LocalNotification"

Both plugins work fine alone, but added together they give the build error.

duplicate symbol _UIApplicationRegisterUserNotificationSettings in:
    /Users/user/my-mobile-application/platforms/ios/build/myApp.build/Debug-iphonesimulator/myApp.build/Objects-normal/i386/AppDelegate.o
    /Users/user/my-mobile-application/platforms/ios/build/Sparwelt.build/Debug-iphonesimulator/myApp.build/Objects-normal/i386/AppDelegate+APPRegisterUserNotificationSettings.o
ld: 1 duplicate symbol for architecture i386
clang: error: linker command failed with exit code 1 (use -v to see invocation)

So basically, which version is this fixed in?

**Comment by t1tlehto** [comment]
I fixed my build error by commenting out line 27 of 
platforms/ios/MyApp/Plugins/de.appplant.cordova.common.registerusernotificationsettings/AppDelegate+APPRegisterUserNotificationSettings.m

//NSString* const UIApplicationRegisterUserNotificationSettings = @"UIApplicationRegisterUserNotificationSettings";

**Comment by JoergHolz** [comment]
@t1tlehto: thx, that works for me too.

**Comment by Linerd00** [comment]
@t1tlehto Thanks, it works
